arcgis-ios-sdk-samples/Layers/Group layers/README.md

```diff
@@ -10,13 +10,17 @@ Group layers communicate to the user that layers are related and can be managed
 
 ## How to use the sample
 
-The layers in the map will be displayed in a table of contents. Toggle the switch next to a layer's name to change its visibility. Turning a group layer's visibility off will override the visibility of its child layers.
+The layers in the map will be displayed in a table of contents. Toggle the switch next to a layer's name or tap the cell to change its visibility. Turning a group layer's visibility off will override the visibility of its child layers.
```

Comment:
> Toggle the switch next to a layer's name or tap the cell to change its visibility.

That kind of makes it sound like there's two different ways to change a layer's visibility. You might consider changing it to something like this: "For project area group layers, toggle the switch next to the layers name to change its visibility. For buildings group layers, tap a row to make that layer the visible layer in that group."